elfpy/data/contract_interface.py

```diff
@@ -420,3 +435,87 @@ def get_hyperdrive_config(hyperdrive_contract: Contract) -> PoolConfig:
     out_config["termLength"] = term_length
 
     return PoolConfig(**out_config)
+
+
+def get_wallet_info(
+    hyperdrive_contract: Contract,
+    base_contract: Contract,
+    block_number: BlockNumber,
+    transactions: list[Transaction],
+) -> list[WalletInfo]:
+    """Retrieves wallet information at a given block given a transaction
```

Comment:
We seem to be drifting from adding sphinx documentation from our functions.  lets stop the bleeding and add them to these functions.  We were pretty good about this for awhile, lets get back to it.  Look around the repo for examples.

Reply:
agreed! Thanks for this Matt, we all need to be better about it.

Reply:
Sure, a lot of these functions were not written by me, but I'll add docstrings to the functions I've touched.